src/pages/dailyReport.js

```diff
@@ -0,0 +1,152 @@
+            window.location.hash = '#';
+        }
+    });
+}
+
+
+export const renderDailyReport = async () => {  
+    let template = `
+            <div class="container">
+            <div class="row">
+                <div class="col-lg-2" style="margin-bottom:20px">
+                    <h2>Reports</h2>
+                    ${reportSideNavBar()}
+                </div>
+                <div class="col-lg-10">
+                    <div class="row">
+                    ${renderCollectionLocationList()}
+                        <div class="col-lg">
+                            <table id="populateDailyReportTable" class="table table-bordered">
+                            </table>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>`;
+    document.getElementById('contentBody').innerHTML = template;
+    removeActiveClass('nav-link', 'active');
+    const navBarBtn = document.getElementById('navBarDailyReport');
+    navBarBtn.classList.add('active');
+    initializeDailyReportTable();
+}
+
+const renderCollectionLocationList = () => {
+    let template = ``;
+    template += `       
+            <div style="margin-top:10px; padding:15px;" class="dropdown">
+                <button class="btn btn-secondary dropdown-toggle dropdown-toggle-sites" id="dropdownSites" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Filter by Collection Location
+                </button>
+                <ul class="dropdown-menu scrollable-menu" id="dropdownMenuButtonSites" aria-labelledby="dropdownMenuButton">
+                    <li><a class="dropdown-item" data-siteKey="all" id="all">All</a></li>
+                    <li><a class="dropdown-item" data-siteKey="ucDcam" id="ucDcam">UC-DCAM</a></li>
+                    <li><a class="dropdown-item" data-siteKey="marshfield" id="marshfield">Marshfield</a></li>
+                    <li><a class="dropdown-item" data-siteKey="weston" id="weston">Weston</a></li>
+                    <li><a class="dropdown-item" data-siteKey="lakeHallie" id="lakeHallie">Lake Hallie</a></li>
+                    <li><a class="dropdown-item" data-siteKey="hpRC" id="hpRC">HP Research Clinic</a></li>
+                    <li><a class="dropdown-item" data-siteKey="hfhKRC" id="hfhKRC">HFH K-13 Research Clinic</a></li>
+                    <li><a class="dropdown-item" data-siteKey="hfhPRC" id="hfhPRC">HFH Cancer Pavilion Research Clinic</a></li>
+                    <li><a class="dropdown-item" data-siteKey="hfhLRC" id="hfhLRC">HFH Livonia Research Clinic</a></li>
+                    <li><a class="dropdown-item" data-siteKey="sfImag" id="sfImag">Sioux Falls Imagenetics</a></li>
+                    <li><a class="dropdown-item" data-siteKey="ingHar" id="ingHar">Ingalls Harvey</a></li>
+                    <li><a class="dropdown-item" data-siteKey="rivEas" id="rivEas">River East</a></li>
```

Comment:
feels like this could be automated via looping in the future so we can eliminate extra places that need updates if sites are added.. fine for now